shopfloor_reception/services/reception.py

```diff
@@ -370,11 +370,30 @@ def _set_quantity__by_package(self, picking, selected_line, barcode):
                         picking, selected_line, message=message
                     )
                 else:
+                    quantity = selected_line.qty_done
+                    new_line, qty_check = selected_line._split_qty_to_be_done(
```

Comment:
I cannot find where new_line is used?

Reply:
It's not.

Reply:
Then
```suggestion
                    __, qty_check = selected_line._split_qty_to_be_done(
```
No idea why pre-commit isn't saying anything about this.

Reply:
Hmmm I originally kept `new_line` because we were using _.("blabla") directly somewhere in the reception scenario, which caused issues with Python. I'm checking and it looks like we moved all those occurrences to the message.py file, so it should be ok to use _. I'll update it.

Reply:
it is a double underscore :)
Not the same thing as the single one that odoo uses for translating things :wink:

Reply:
Well I remember there was a reason why we kept `new_line`. I've tested it with underscore and it seems to work fine so I'll keep it.

Reply:
You mean double underscore?

Reply:
Yup